### DISCUSSION TITLE
Circle movement in sliders.

### DISCUSSION THREAD

**Discussion by defreeyay** [ppy/osu#21734] [Q&A] [answered]
Hello! I don’t know how to use github, but I hope that this message will reach here.
Yesterday, when i watched the stream of the top player talala of gameplay lazer, i noticed that the sliders are still moving strangely. I thought that at least someone would notice and write to the github, but apparently not. I noticed that the sliders are very different from the stable visually and that is certainly noticeable and annoying because it looks crooked and not pleasant. Is this such an idea? I really don't understand this. This greatly hinders. I noticed this thing for two years, but I thought that it would be fixed in the end.

https://user-images.githubusercontent.com/121093587/208638358-9847bae3-ee43-4162-8387-142827f7a8c3.mp4

You may notice how the slider circle sometimes moves and sometimes stays in place. In the stable the movement of the circle is completely absent.

**Comment by PercyDan54** [comment, marked as answer]
Do you mean:
https://github.com/ppy/osu/wiki/Gameplay-differences-from-osu!stable#slider-head-circle-follows-track-snaking-when-not-hit

**Comment by defreeyay** [comment]
yes, thank you O_O
